IBPSA/package.mo

```diff
@@ -486,7 +486,9 @@ annotation (
 version="3.0.0",
 versionDate="2018-09-28",
 dateModified = "2018-09-28",
-uses(Modelica(version="3.2.3")),
+uses(Modelica(version="3.2.3"),
+    CONTAM_Modelica(version="2"),
```

Comment:
please remove the dependencies on CONTAM_Modelica and IDEAS.

Reply:
is there a quick check to make sure all dependencies are gone? Do you know if Dymola updates this code accordingly if all dependencies are gone?

Reply:
I'm pretty sure that dymola does not update it. I would do a search using grep (linux) on the word 'CONTAM' in the library directory to figure out whether or not the library is still referenced.